### ISSUE TITLE
monitor: Sends RPL_MONOFFLINE + RPL_MONONLINE when changing capitalization

### ISSUE THREAD

**Issue by progval** [pirc-pl/unrealircd-modules#16]
Hi,

When users use the `NICK` command to change their name in a way that is equivalent modulo case-insensitivity, this module sends RPL_MONOFFLINE (731) + RPL_MONONLINE (730) to users who monitored the nick, instead of being silent:

```
1625304189.857 1 -> S: MONITOR + qux
1625304189.858 S -> 1: :My.Little.Server 731 bar :qux

[...]

1625304189.946 2 -> S: NICK qux
1625304189.946 S -> 2: :baz NICK :qux
1625304189.946 S -> 1: :My.Little.Server 730 bar :qux!~username@Clk-4D552FD5

1625304189.947 2 -> S: NICK QUX
1625304189.947 S -> 2: :qux NICK :QUX
1625304189.988 S -> 1: :My.Little.Server 731 bar :qux
1625304189.988 S -> 1: :My.Little.Server 730 bar :QUX!~username@Clk-4D552FD5
```

I don't know if this is a bug, but this behavior differs from other IRCds I tested (Chary/Solanum, InspIRCd, Ergo).
Feel free to close this if you don't think that's an issue.

**Comment by k4bek4be** [comment]
Fixed in 3334ff9be4b89d515c5c04ba40285ed9e61650aa

**Comment by progval** [comment]
Thanks!
